pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -68,6 +69,21 @@ type cniPortMapping struct {
 	HostIP        string `json:"hostIP"`
 }
 
+// cniBandwidthEntry maps to the standard CNI bandwidth Capability
+// see: https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md
+type cniBandwidthEntry struct {
+	// IngressRate is the bandwidth rate in Kbps for traffic through container. 0 for no limit. If ingressRate is set, ingressBurst must also be set
+	IngressRate int `json:"ingressRate"`
```

Comment:
All of these fields should have `omitEmpty`

Reply:
Done.